SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -ex
+
+parse_cmdline_args() {
+    # Ensure that the 'dracut-lib' is present and loaded.
+    type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
+
+    VERITY_MOUNT="mnt/verity_mnt"
```

Comment:
is it intentional that VERITY_MOUNT does not have leading `/`? i.e. should it be '/mnt/verity_mnt`?